mesheryctl/internal/cli/root/model/model.go

```diff
@@ -196,8 +196,7 @@ func prettifyJson(model model.ModelDefinition) error {
 func listModel(cmd *cobra.Command, url string, displayCountOnly bool) error {
 	req, err := utils.NewRequest(http.MethodGet, url, nil)
 	if err != nil {
-		utils.Log.Error(err)
-		return nil
+    return err
```

Comment:
```suggestion
    utils.Log.Error(err)
    return nil
```

Reply:
I think this would print the error 2 times.

Suggestion would be only one should be present either return error or utils.log

Reply:
Suggestion should have the same behavior on output and follow the same behavior done in all mesheryctl commands for example https://github.com/meshery/meshery/blob/492dd8b75982038cc2ddc843df9c8e3eed88a8a0/mesheryctl/internal/cli/root/registry/publish.go#L130

Reply:
@lekaf974 commited the suggested changes